connector/tracetotopoconnector/internal/mapping_test.go

```diff
@@ -22,10 +22,18 @@ func TestMapping_MapComponent(t *testing.T) {
 	testSpan.Attributes().PutStr("kind", "licence")
 	testSpan.Attributes().PutStr("priority", "urgent")
 
+	testScope := ptrace.NewScopeSpans()
+	testScope.Scope().Attributes().PutStr("name", "kamon")
+
+	testResource := ptrace.NewResourceSpans()
+	testResource.Resource().Attributes().PutStr("name", "microservice")
```

Comment:
In a real situation the `env.name` and `service.name` attributes should be resource attributes, but we can easily align that when we update the tests again next time.